### ISSUE TITLE
Will it be updated for version 3 of Vuejs ?

### ISSUE THREAD

**Issue by laurent-brisbois** [kutlugsahin/vue-smooth-dnd#157]
Hello !

Vuejs version 3 will come soon and I wanted to know if it is planned to update vue-smooth-dnd accordingly ?

**Comment by ivictbor** [comment]
@laurent-brisbois I am afraid the issue is currently bigger then Vue3 support because it is not maintained for a year

**Comment by tstecca** [comment]
We're moving off of this library.  It doesn't appear to be maintained.  It was a good ride.  Thanks to kutlugsahin for their efforts over the years.

**Comment by laurent-brisbois** [comment]
Yes this is "sad".. Because I didn't find any equivalent. I mean there is vue-draggable but I don't like the fact that it is only a shadow copy of the element that is dragged...

**Comment by kolaente** [comment]
@tstecca Have you found a good replacement?

**Comment by ivictbor** [comment]
@tstecca would also love to hear about your experience. I've been searching for a while for such library for kanban board implementation for https://tracklify.com, and found that @kutlugsahin work is still the best in this area for our needs, except having several minor issues with jumps/and total mobile freeze, it still works good. We also planned to fix several issues by ourselves but it is still in todo as well ;)

**Comment by tstecca** [comment]
We've migrate half of our stuff to https://www.npmjs.com/package/vuedraggable.  It's got 283,312 weekly downloads and is updated regularly.  It's based off of the popular https://www.npmjs.com/package/sortablejs package which is very popular.  Sortable.js is the Framework-agnostic package the handles the dragging, while Vue-Draggable is the Vue implementation of it.  There are also packages for React-Draggable and Angular-Draggable.

We didn't have any troubles migrating.  It's a good package.  I didn't do it myself, it was another guy on the team, but I know it went well.

**Comment by gilnd** [comment]
Hey there, i made a port to Vue 3, [see more here](https://github.com/gilnd/vue3-smooth-dnd) :)
